java/java.editor/src/org/netbeans/modules/editor/java/GoToSupport.java

```diff
@@ -986,6 +1020,14 @@ public Void visitTypeParameter(TypeParameterElement e, Boolean highlightName) {
             return null;
         }
         
+        @Override
+        public Void visitUnknown(Element e, Boolean p) {
+            if (TreeShims.isRecordComponent(e)) {
+                return visitVariable((VariableElement) e, p);
+            }
+            return super.visitUnknown(e, p); //To change body of generated methods, choose Tools | Templates.
```

Comment:
Ideally comments like "//To change..." should be deleted.

Reply:
Comment addressed.